forestci/calibration.py

```diff
@@ -47,62 +46,67 @@ def gfit(X, sigma, p=5, nbin=200, unif_fraction=0.1):
     sigma: float
         Noise estimate on X.
     p: int
-        Number of parameters used to fit G. Default: 5
+        Number of parameters used to fit G.
     nbin: int
         Number of bins used for discrete approximation.
-        Default: 200
     unif_fraction: float
-        Fraction of G modeled as "slab". Default: 0.1
+        Fraction of G modeled as "slab".
 
     Returns
     -------
     An array of the posterior density estimate g.
     """
-    min_x = min(min(X) - 2 * np.std(X, ddof=1), 0)
+    min_x = max(min(X) - 2 * np.std(X, ddof=1), 0)
```

Comment:
no need to model the prior on negative values -- we are fitting a nonnegative variable!